src/Bannerlord.Diplomacy/Messengers/MessengerManager.cs

```diff
@@ -213,9 +218,14 @@ private TextObject GetMessengerSentText(IFaction faction1, IFaction faction2, He
         {
             var textObject =
                 new TextObject(
-                    "{=qNWMZP0z}The messenger from {FACTION1_NAME} will arrive at {HERO_NAME} of {FACTION2_NAME} within {TRAVEL_TIME} days.");
+                    "{=qNWMZP0z}The messenger from {FACTION1_NAME} will arrive at {HERO_NAME} of {FACTION2_TEXT} within {TRAVEL_TIME} days.");
```

Comment:
```suggestion
                    "{=qNWMZP0z}The messenger from {FACTION1_NAME} will arrive at {HERO_NAME} {FACTION2_TEXT} within {TRAVEL_TIME} days.");
```
Otherwise it will show e.g. as 'of of Vlandia' I think.